packages/eslint-plugin/src/rules/unbound-method.ts

```diff
@@ -18,6 +18,61 @@ export type Options = [Config];
 
 export type MessageIds = 'unbound';
 
+const nativelyBoundMembers = [
+  ...[
+    Promise,
+    //    BigInt, // todo: enable when we drop node@8
+    Number,
+    Object,
+    String,
+    RegExp,
```

Comment:
nit - `RegExp` exposes no methods

Reply:
Very good point - I got confused by all the mutated properties it has 🙄